storage/types.go

```diff
@@ -362,13 +362,21 @@ func (n *NodeID) MaskLeft(depth int) *NodeID {
 }
 
 // Neighbor returns the same node with the bit at PrefixLenBits flipped.
+// In terms of a tree traversal, this is the parent node's other child node
+// in the binary tree (often termed sibling node).
 func (n *NodeID) Neighbor() *NodeID {
 	height := n.PathLenBits() - n.PrefixLenBits
 	n.FlipRightBit(height)
 	return n
 }
 
 // Siblings returns the siblings of the given node.
+// In terms of a tree traversal, this returns the Neighbour() of every node
+// (including this one) on the path up to the root. The array is of length
+// PrefixLenBits and is ordered such that the nodes closest to the leaves are
+// earlier in the array.
+// These nodes are the ones that would be required for a Merkle proof for this
```

Comment:
Clarify the kind of proof:
`... required for a Merkle tree inclusion proof ...` ?